MANIFEST.in

```diff
@@ -7,6 +7,7 @@ include src/pyird/data/samples/aprefC
 include src/pyird/data/samples/apref_a
 include src/pyird/data/samples/apref_c
 include src/pyird/data/thar_mmf2_a_H_20210317.fits
+include src/pyird/data/samples/flat_sample_m2.dat
 
```

Comment:
Please add
`include src/pyird/data/samples/wflat_m2_20210317.dat`
here

Reply:
Once it was added, I confirmed that all of the unit tests was passed.

Reply:
Sorry, I have mistaken the file name. I'll correct it.